### PR TITLE
Add elevation property for BpkDividedCard SwiftUI version

### PR DESCRIPTION
**Pull request by iHandle**
**Changes**
* Extract the `Elevation` enum outside of the `BPKCard` as `BPKCardElevation` so that the Divided Card can use it also.
* Add `elevation` property for BpkDividedCard SwiftUI version.
* Updat unit tests, the example screenshot and the README.md.

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `README.md`
+ [x] Tests
+ [ ] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [ ] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift [hunk 1]
```diff
@@ -19,20 +19,23 @@
...
     private let primaryContent: PrimaryContent
     private let secondaryContent: SecondaryContent
     private var tapAction: () -> Void = {}
 
     public init(
+        isElevated: Bool = true,
         @ViewBuilder primaryContent: () -> PrimaryContent,
         @ViewBuilder secondaryContent: () -> SecondaryContent
     ) {
+        self.isElevated = isElevated
```
Comment (gert-janvercauteren) [inline Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift:32]: Can you please use the same Enum for the elevation option? 
We can basically 'copy' the way BPKCard does it.

**Review by gert-janvercauteren — COMMENTED** [review]
Can we update the background color of the page, so we can see the corners of the non elevated card? 
(Same for the screenshot tests)

File: Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift (hunk 1 above)
Comment (iHandle) [reply Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift:32]: Can we just use `BPKCard.Elevation` directly? If we define an elevation Enum inside `BPKDividedCard`, we have to convert `BPKDividedCard.Elevation` to `BPKCard.Elevation`.

Comment (gert-janvercauteren) [reply Backpack-SwiftUI/Card/Classes/BPKDividedCard.swift:32]: Yes we can use it directly.

**Comment by iHandle** [comment]
> 

If we set the background color as, for example blue, It's hard to tell the difference between elevated card and non elevated card.
image

**Review by gert-janvercauteren — APPROVED** [review]
LGTM, thanks for contributing!

**Pull request merged**